src/scripts/settings/webSettings.js

```diff
@@ -85,6 +85,15 @@ export function getMultiServer() {
     });
 }
 
+export function getServers() {
+    return getConfig().then(config => {
+        return config.servers;
```

Comment:
In case of missing of `servers` field in config file.
```suggestion
        return config.servers || [];
```